pkg/products/rhsso/reconciler.go

```diff
@@ -530,10 +514,6 @@ func syncronizeWithOpenshiftUsers(keycloakUsers []*aerogearv1.KeycloakUser, ctx
 		return nil, err
 	}
 	for _, kcUser := range keycloakUsers {
-		if kcUser.UserName == CustomerAdminUser.UserName {
-			continue
-		}
-
 		kcUser.ClientRoles = getKeycloakRoles(isOpenshiftAdmin(kcUser, openshiftAdminGroup))
```

Comment:
`isOpenshiftAdmin(kcUser, openshiftAdminGroup)` can be removed since now it's always the same roles that get assiged to every user